workspaces/adventure-pack/goodies/java/src/integer_digits/AP.java

```diff
@@ -0,0 +1,30 @@
+package integer_digits;
+
+import iterable_int_stream.IterableIntStream;
+import java.util.stream.IntStream;
+
+public final class AP {
+
+  /**
+   * Get the digits in the given integer. For example, in the given integer 8725, digits are
+   * [8, 7, 2, 5].
+   * @param number Any number of interest.
+   * @return The {@link IterableIntStream stream} of digits in the given integer.
+   */
+  public static IterableIntStream integerDigits(int number) {
+    var digitsInReverse = IntStream.iterate(
+      number,
+      n1 -> n1 != 0,
```

Comment:
I'm not sure whether this means that for an input of 0, there are no digits. It's worth having 0 as a test case when we add the unit test.

Reply:
Comment Addressed.